de.dlr.sc.virsat.ide.product/virsat_ide.product

```diff
@@ -79,23 +79,18 @@ This product includes software developed by other open source projects including
       <feature id="org.eclipse.e4.core.tools.feature" installMode="root"/>
       <feature id="org.eclipse.eclemma.feature" installMode="root"/>
       <feature id="org.eclipse.egit" installMode="root"/>
-      <feature id="org.eclipse.egit.mylyn" installMode="root"/>
-      <feature id="org.eclipse.m2e.feature" installMode="root"  version="1.11.0.qualifier"/>
-      <feature id="org.eclipse.m2e.logback.feature" installMode="root"  version="1.11.0.qualifier"/>
-      <feature id="org.eclipse.mylyn.bugzilla_feature" installMode="root"/>
-      <feature id="org.eclipse.mylyn.context_feature" installMode="root"/>
-      <feature id="org.eclipse.mylyn.gerrit.feature" installMode="root"/>

```

Comment:
Where are the mylyn features? Shouldn't they still be part of our IDE?

Reply:
Readded the features to the product, unclear why the were removed in the first place.